lib/ansible/module_utils/network/aci/aci.py

```diff
@@ -446,7 +630,7 @@ def construct_url(self, root_class, subclass_1=None, subclass_2=None, subclass_3
 
         if self.child_classes:
             # Append child_classes to filter_string if filter string is empty
-            self.update_qs({'rsp-subtree': 'full', 'rsp-subtree-class': ','.join(self.child_classes)})
+            self.update_qs({'rsp-subtree': 'full', 'rsp-subtree-class': ','.join(sorted(self.child_classes))})
```

Comment:
Why sorted?

Reply:
When under test, self.child_classes appears to randomize the response when .join() is called (the unit test randomly fails). Sorted() eliminates the randomization and allows the test to pass consistently. My next PR will allow aci.py to be testable and I will be able to demonstrate this with the unit tests I have ready to upload.